src/components/Garden/Feed/ProposalInfo.tsx

```diff
@@ -42,6 +50,9 @@ function ProposalInfo({ loading, proposal, onSelectProposal }) {
             decimals={primaryToken.decimals}
             icon={primaryToken.icon}
             symbol={primaryToken.symbol}
+            verified={undefined}
+            color={undefined}
+            size={undefined}
           />
```

Comment:
Why we need undefined here? To clear them out?

Reply:
Yep. It's fixed now 👌